src/pages/Authentication/Login.jsx

```diff
@@ -22,65 +21,65 @@ export default function Login() {
 
   const handleLogin = (e) => {
     e.preventDefault()
-    sendLoginData(loginData)
+    submitLoginData(loginData)
   }
 
-  const sendLoginData = async (loginData) =>{
+  const submitLoginData = async (loginData) =>{
     try {
       const response = await axios.post("/api/auth/login", loginData)
-      console.log(response)
       if (response.status === 200){ 
-      const {data} = response;
-      const userToken = data.encodedToken
-      localStorage.setItem("userToken", userToken)
-      navigate("/products")
+        const userToken = response.data.encodedToken
+        localStorage.setItem("userToken", userToken)
+        navigate("/products")
       }
     } catch (error) {
       setLoginError("An error occurred.")
       console.log("error:", error)
     }
   }
 
+  const guestLogin = () => {
+    submitLoginData(testUser)
+  }
+
   return (
     <div className='page-body'>
       <section className="login-container shadow">
         <h2 className="container-title">Login</h2>
         <form className="login-details" onSubmit={handleLogin}>
-            <div className="labelled-input label-top username">
-                <label className="label"> Email ID</label>
-                <input type="email" name="email" placeholder="jane.doe@email.com" onChange={setUserData}/>
-            </div>
-            <div className="labelled-input label-top password">
-                <label className="label"> Password </label>
-                <span className="toggle-login-pwd material-icons material-icons-outlined" onClick={()=>setShowPwd(!showPwd)}>
-                {showPwd ? "visibility_off" : "visibility"}
-                </span>
-                <input type={showPwd ? "text" : "password"} name="password" placeholder="************" onChange={setUserData}/>
-            </div>
-            <div className="auth-opts">
-                <div className="checkbox-btn remember-me">
-                    <input type="checkbox" name="checkbox"/>
-                    <label className="label">Remember Me</label>
-                </div>
-                <a href="#" className="forgot-pwd">Forgot Password?</a>
+          <div className="labelled-input label-top username">
+            <label className="label"> Email ID</label>
+            <input type="email" name="email" placeholder="jane.doe@email.com" onChange={setUserData}/>
+          </div>
+          <div className="labelled-input label-top password">
+            <label className="label"> Password </label>
+            <span className="toggle-login-pwd material-icons material-icons-outlined" onClick={()=>setShowPwd(!showPwd)}>
+            {showPwd ? "visibility_off" : "visibility"}
+            </span>
+            <input type={showPwd ? "text" : "password"} name="password" placeholder="************" onChange={setUserData}/>
```

Comment:
instead of using ************ as a placeholder you could have written Enter Password it will make a better UX